BugTriaging.md

```diff
@@ -0,0 +1,43 @@
+# Teku Bug Triage Process
+
+In Teku, we have a well-defined policy regarding bug categorization. In a nutshell, our bugs are categorized into 5 priorities. 
+
+#### Bug Priorities
+
+| Priority | Examples |
+|-|-|
+| Very High (P1) | <ul><li>Affects consensus</li> <li>Breaks sync</li><li>Security issue</li></ul> |
+| High (P2) | <ul><li> Degrades validator rewards</li> <li>Unexpected behaviour of core features</li></ul>|
```

Comment:
I'd add "or incurs penalties" to the validator rewards point and add "Significant performance regression" or maybe just "Performance degradation that affects ability to participate in the network"

Reply:
Done.